cardano-cli/src/Cardano/CLI/Helpers.hs

```diff
@@ -101,6 +106,22 @@ ensureNewFile writer outFile blob = do
 ensureNewFileLBS :: FilePath -> ByteString -> ExceptT HelpersError IO ()
 ensureNewFileLBS = ensureNewFile BS.writeFile
 
+bounded :: forall a. (Bounded a, Integral a, Show a) => String -> Opt.ReadM a
```

Comment:
Where is this used?

Reply:
Here: https://github.com/IntersectMBO/cardano-cli/blob/25fcdff827c111e9bbddebfc679dc144cf96dd35/cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs#L84